llama.cpp

```diff
@@ -14977,6 +14987,14 @@ void llama_free_model(struct llama_model * model) {
     delete model;
 }
 
+void llama_model_kv_override_free(struct llama_model_kv_override * kv_overrides) {
+    for (const struct llama_model_kv_override *p = kv_overrides; p->key[0] != 0; p++) {
```

Comment:
@ggerganov Added, but I worry where to call it ? can it be included in `llama_model_free` or all common examples must be changed ?

Reply:
@slaren, do you have an idea on this, please ? Where the kv overrides must be freed ?

Reply:
This cannot be done in llama.cpp (core). The memory is allocated by the caller, and needs to be freed by the caller. Presumably after calling `llama_load_model_from_file`.